package.json

```
@@ -16,7 +16,7 @@
     "react": "^16.13.1",
     "react-alert": "^7.0.0",
     "react-bootstrap": "^1.0.0",
-    "react-color": "^2.17.3",
+    "react-color": "^2.18.0",
```

```suggestion
    "react-color": "^2.17.3",
```
The older version was there for a reason 😅.